hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/hudi/HoodieMergeOnReadRDD.scala

```diff
@@ -18,6 +18,8 @@
 
 package org.apache.hudi
 
+import java.util.Properties
```

Comment:
nit: this import should be in the next group

Reply:
Fixed!